datumaro/plugins/openvino/README.md

```diff
@@ -43,30 +50,31 @@ Interpreter samples to parse OpenVINO inference outputs.
 
 ## Model inference
 - Prerequisites: 
-    - Put the models downloaded in <datumaro_root_dir>/datumaro/plugins/openvino/models
-    - Put the VOCdevkit dataset downloaded in <datumaro_root_dir>
+    - OpenVINO™ (To install OpenVINO™, please see the [OpenVINO™ Installation Instruction](https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_installing_openvino_linux.html)
+    - Datumaro (To install Datumaro, please see the [User Manual](docs/user_manual.md)
+    - OpenVINO™ models (To download OpenVINO™ models, please see the [Model Downloader Instruction] https://docs.openvinotoolkit.org/latest/omz_tools_downloader_README.html)
+    - PASCAL VOC 2012 dataset (To download VOC 2012 dataset, please go [VOC2012 download](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/#devkit)
     
-- To run the inference with OpenVINO models and the interpreter samples, please follow the instructions below.
+- To run the inference with OpenVINO™ models and the interpreter samples, please follow the instructions below.
 
   ```bash
-  # . <openvino_dir>/bin/setupvars.sh
-  # cd <datumaro_root_dir>/datumaro/plugins/openvino/
+  # <openvino_dir>/bin/setupvars.sh
   #	datum create -o <proj_dir>
   #	datum model add -l <launcher> -p <proj_dir> --copy -- -d <path_to_xml> -w <path_to_bin> -i <path_to_interpreter_script>
   #	datum add path -p <proj_dir> -f <format> <path_to_dataset>
   #	datum model run -p <proj_dir> -m model-0
   #
   # Examples
   # Detection> ssd_mobilenet_v2_coco
-  . /opt/intel/openvino_2021/bin/setupvars.sh
+  /opt/intel/openvino_2021/bin/setupvars.sh
```

Comment:
`.` or `source` in the beginning is required

Reply:
If we install openvino 2021.r1, the default path is /opt/intel/openvino_2021, not /opt/intel/openvino. The default path like openvino_2021 had been using since openvino 2021.r1.  Do you still want me to change it?

Reply:
I changed it to openvino because we don't know user which Openvino version will install. openvino is more general. So I changed it.